content/800-data-platform/850-billing/04-manage-plans.mdx

```diff
@@ -67,16 +67,56 @@ You are redirected to chat with a representative.
+If the **Scale** plan no longer makes sense for a project, you can downgrade the project to the **Free** plan.
+
+<Admonition>
+
+**Note**:<br /><br />
+
+If you need to downgrade a project that is on the **Enterprise** plan, [contact Support](/data-platform/contact-support).
+
+</Admonition>
+
+Downgrading to the **Free** plan includes a number of changes to your project.
+
+- If you have more team members or environments than are included in the quota on the **Free** plan, you must reduce the number of existing team members or environments.
+- The Data Proxy CPU usage changes to the hard limit included with the **Free** plan.
+- All team members receive the Admin role. The roles Viewer, Collaborator, and Developer are not available on the **Free** plan.
+
+When you downgrade a project to the **Free** plan, we will issue a refund for the remaining time until the end of the month for which you still have unused overages on extra team members or environments.
+
+For example, if on the **Scale** plan you have one extra member the overage for which you have pre-paid at the beginning of the month, for the remainder of the current month on that overage we will give you a refund.
+
+If your project has Data Proxy CPU milliseconds overages until the downgrade, the invoice for your next month will include the charges for those.
+
+**Prerequisites**
+
+Learn about the [**Free** plan](/data-platform/billing/plans-and-quotas#strongfreestrong-plan) and the quotas that it includes for team members, environments, and Data Proxy CPU usage.
 
 **Steps**
 
-1. Go to the **[Billing](https://cloud.prisma.io/billing)** page.
-2. Under _Your projects_, click **Cancel** for the project that you want to downgrade.
-3. In the chat, submit your downgrade request to the Support team.
+1. Go to the _[Billing](https://cloud.prisma.io/billing)_ page.
+2. Under _Your projects_, click **Manage plan** for the project that you want to downgrade.
+3. Select the **Free** plan and click **Downgrade**.
+   <img
+     src="https://res.cloudinary.com/prismaio/image/upload/v1665489638/docs/pdp/manage-project-plan-select-free-and-click-downgrade_d2jzb6.png"
+     alt="Prisma Data Platform - Manage project plan - Select Free plan and click Downgrade"
+   />
+4. Click **Confirm downgrade** if the number of collaborators or environments on your project does not exceed the quotas on the [**Free** plan](https://www.prisma.io/docs/data-platform/billing/plans-and-quotas#strongfreestrong-plan).
+5. If on your project you have more collaborators or environments than are included in the quotas on the [**Free** plan](https://www.prisma.io/docs/data-platform/billing/plans-and-quotas#strongfreestrong-plan), decide how to handle each overage.
+   1. Expand _Team members and_ select the allowed number of team members you want to keep on the project.
+   2. Expand _Environments_ and do the same for the environments you want to keep on the project.
```

Comment:
Would it be helpful to add a comment here about the default environment?
You won't be able to remove it from this form, since it's required for a project to have one default.
So if you want to delete it, you'll have to first assign a new default environment